aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -153,6 +153,40 @@ For example, an image asset in the *wwwroot/images* folder is accessible to the
 
 [Multi-value cookies](xref:System.Web.HttpCookie.Values) aren't supported in ASP.NET Core. Create one cookie per value.
 
+## Partial app migration
+
+One approach is to create an IIS sub-application and only move certain routes from ASP.NET 4.x to ASP.NET Core while preserving the URL structure the app. For example, consider the relevant section from the *applicationHost.config* file:
```

Comment:
URL structure _of_ the app

Reply:
@bariscaglar see my change. If that's not right, please suggest the entire line I can add.